app/download/views.py

```diff
@@ -37,6 +37,49 @@ def download_document(service_id, document_id):
     except ValueError:
         return jsonify(error="Invalid decryption key"), 400
 
+    try:
+        check_scan_verdict(service_id, document_id, sending_method)
+    except (MaliciousContentError, SuspiciousContentError) as e:
+        current_app.logger.info(
+            "Malicious content detected, refused to download document: {}".format(e),
+            extra={
+                "service_id": service_id,
+                "document_id": document_id,
+            },
+        )
+        return jsonify(error=str(e)), MALICIOUS_CONTENT_ERROR_CODE
+    except ScanInProgressError as e:
+        age_seconds = scan_files_document_store.get_object_age_seconds(
+            service_id, document_id, sending_method
+        )
+        if age_seconds > SCAN_TIMEOUT_SECONDS:
+            current_app.logger.info(
+                "Scan timed out for document: {}".format(e),
+                extra={
+                    "service_id": service_id,
+                    "document_id": document_id,
+                },
+            )
+            return jsonify(scan_verdict="scan_timed_out"), 200
+
+        current_app.logger.info(
+            "Scan in progress, refused to download document: {}".format(e),
+            extra={
+                "service_id": service_id,
+                "document_id": document_id,
+            },
+        )
+        return jsonify(error=str(e)), SCAN_IN_PROGRESS_ERROR_CODE
```

Comment:
```suggestion
            pass

        else:
            current_app.logger.info(
                "Scan in progress, refused to download document: {}".format(e),
                extra={
                    "service_id": service_id,
                    "document_id": document_id,
                },
            )
            return jsonify(error=str(e)), SCAN_IN_PROGRESS_ERROR_CODE
```

Reply:
Sorry I didn't catch this in my first review - I think we need this logic in both document download endpoints so that the document will be returned if the scan has timed out